src/set.js

```diff
@@ -0,0 +1,92 @@
+SVG.Set = SVG.invent({
+  create: function (members) {
+    this.length = members.length
+    for (var i = members.length - 1; i >= 0; --i) {
+      this[i] = members[i]
+    }
+  }
+  // Add class methods
+, extend: {
+    // Add element to set
+    add: function() {
+      Array.prototype.push.apply(this, arguments)
+      return this
+    }
+    // Remove element from set
+  , remove: function(element) {
+      var i = this.index(element)
+
+      // remove given child
+      if (i > -1)
+        Array.prototype.splice.call(this, i, 1)
+
+      return this
+    }
+  , each: function (block, value) {
```

Comment:
Its unclear to me what block should be if it isn't a function?
Can you clarify?

Reply:
it can be a property of an element, too. I will rename it to property to make the intention more clear